cocos/ui/view.ts

```diff
@@ -140,7 +140,6 @@ export class View extends Eventify(System) {
 
         // For now, the engine UI is adapted to resolution size, instead of window size.
         screen.on('window-resize', this._updateAdaptResult, this);
-        screen.on('orientation-change', this._updateAdaptResult, this);
```

Comment:
Doesn't need to listen on `orientation-change` here now ?

Reply:
Developer can import screen from cc and then register event by screen